frontend/src/components/ContributorAvatar.tsx

```diff
@@ -38,9 +38,11 @@ const ContributorAvatar = memo(({ contributor, uniqueKey }: ContributorProps) =>
       ? ` in ${(contributor as TopContributorsTypeGraphql).projectName}`
       : ''
 
+  const projectName = !isAlgolia ? (contributor as TopContributorsTypeGraphql).projectName : null
+
   return (
     <Tooltip
-      id={`avatar-tooltip-${login}-${uniqueKey}`}
+      id={`avatar-tooltip-${login}-${uniqueKey}-${projectName || 'unknown-project'}`}
```

Comment:
But this always falls back to `unknown-project` as Algolia response never contains `projectName`.